lib/features/payment/payment_state.dart

```diff
@@ -1,102 +1,158 @@
 import 'package:dap/dap.dart';
-import 'package:decimal/decimal.dart';
-import 'package:didpay/features/countries/countries.dart';
 import 'package:didpay/features/pfis/pfi.dart';
 import 'package:didpay/features/transaction/transaction.dart';
 import 'package:tbdex/tbdex.dart';
+import 'package:web5/web5.dart';
 
 class PaymentState {
   final TransactionType transactionType;
-  final String? payinCurrency;
-  final String? payoutCurrency;
-  final String? serviceFee;
-  final String? paymentName;
-  final String? exchangeId;
-  final Decimal? payinAmount;
-  final Decimal? payoutAmount;
-  final Decimal? exchangeRate;
-  final Dap? dap;
-  final List<MoneyAddress>? moneyAddresses;
-  final Country? selectedCountry;
-  final Pfi? selectedPfi;
-  final Offering? selectedOffering;
-  final PayinMethod? selectedPayinMethod;
-  final PayoutMethod? selectedPayoutMethod;
-  final List<PayinMethod>? payinMethods;
-  final List<PayoutMethod>? payoutMethods;
-  final List<String>? claims;
-  final Map<String, String>? formData;
-  final Map<Pfi, List<Offering>>? offeringsMap;
+  final Rfq? _rfq;
+  final Offering? _offering;
+  final GetOfferingsFilter? _getOfferingsFilter;
+  final Pfi? _pfi;
+  final Dap? _dap;
+  final String? _paymentName;
+  final List<MoneyAddress>? _moneyAddresses;
+  final Map<String, String>? _paymentDetails;
+  final Map<Pfi, List<Offering>>? _offeringsMap;
```

Comment:
Is the plan here to keep chipping away at this state as we pull out other smaller states?